libwebauthn/src/webauthn.rs

```diff
@@ -19,6 +19,7 @@ use crate::proto::ctap2::{
     Ctap2, Ctap2ClientPinRequest, Ctap2GetAssertionRequest, Ctap2GetInfoResponse,
     Ctap2MakeCredentialRequest, Ctap2UserVerifiableRequest, Ctap2UserVerificationOperation,
 };
+use crate::transport::error::PlatformError;
```

Comment:
Let's re-export this as per `CtapError`, `Error` and `TransportError` below.

Reply:
Yes, makes sense!